public/app/plugins/panel/graph/module.ts

```diff
@@ -207,16 +207,19 @@ class GraphCtrl extends MetricsPanelCtrl {
       }
     }
 
+    this.render(this.seriesList);
+    this.loading = true;
```

Comment:
Not sure about this. On the one hand it can be confusing to see graph data with a loading indicator still, on the other, folks may wonder where their annotations are if there's no indicator the query is still running. @torkelo thoughts?